fastlane/Fastfile.swift

```diff
@@ -12,11 +12,36 @@ class Fastfile: LaneFile {
 
     // MARK: - Code signing
 
-    func syncCodeSigningLane() {
-        desc("This lane is for development purpose, will be removed after the migration")
-        Match.syncCodeSigning(type: .development, appIdentifier: ["co.nimblehq.ios.templates"])
-        Match.syncCodeSigning(type: .adHoc, appIdentifier: ["co.nimblehq.ios.templates"])
-        Match.syncCodeSigning(type: .appStore, appIdentifier: ["co.nimblehq.ios.templates"])
+    func syncDevelopmentCodeSigningLane() {
+        desc("Sync the development match signing for the staging build")
+        Match.syncCodeSigning(
+            type: .development,
+            appIdentifier: [Constant.stagingBundleId]
+        )
+    }
+
+    func syncAdHocStagingCodeSigningLane() {
+        desc("Sync the ad hoc match signing for the staging build")
+        Match.syncCodeSigning(
+            type: .adHoc,
+            appIdentifier: [Constant.stagingBundleId]
+        )
+    }
+
+    func syncAdHocProductionCodeSigningLane() {
+        desc("Sync the ad hoc match signing for the production build")
+        Match.syncCodeSigning(
+            type: .adHoc,
+            appIdentifier: [Constant.productionBundleId]
+        )
+    }
+
+    func syncAppStoreCodeSigningLane() {
+        desc("Sync the app store match signing for the production build")
```

Comment:
`App Store` should be caps. What do you think we cap `Production` as well so it's easier to glance through?
```suggestion
        desc("Sync the App Store match signing for the Production build")
```

Reply:
I agree

Reply:
Fixed in https://github.com/nimblehq/ios-templates/pull/375/commits/7e94ea9e3b9f85a503859ef4840531c83a51304c